code/modules/psionics/faculties/coercion.dm

```diff
@@ -320,8 +328,9 @@
 			if(!T || L == user || L.stat == DEAD || L.invisibility == INVISIBILITY_LEVEL_TWO)
 				continue
 			if(L.isSynthetic() || L.is_diona() || isvaurca(L))
-				var/obj/item/organ/internal/augment/psiaug = locate() in L
-				if(!psiaug)
+				var/mob/living/carbon/C = target
```

Comment:
This is another crash-risk: `isSynthetic` doesn't necessarily return true for `carbon` mobs. The previous way of doing it was fine.